src/helpers/dateHelpers.ts

```diff
@@ -0,0 +1,39 @@
+export function getCurrentDate(): string {
+  const date = Date(),
+    d = new Date(date),
+    year = d.getFullYear();
+
+  let month = '' + (d.getMonth() + 1),
+    day = '' + d.getDate();
```

Comment:
Are you trying to perform an `abstract operation` here to convert to a string?
If so you can make this more explicit by using the inbuilt `String` helper method.
```ts
let month = String((d.getMonth() + 1)),
  day = String(d.getDate());
```

Reply:
All done. Thanks for pointing that out